File: Photon Tools/Servers/Game/python/scenes/Map_1/Portals/Portal.py
from MathematicsHelper import *
from Game.InterestManagement import *
from Game.Application.SceneObjects import *

position = Vector2(-17.125, -5.5)
playerPosition = Vector2(-12.8, -12.8)
map = 3

portalSceneObject = Portal(position, playerPosition, map)

scene.AddSceneObject(portalSceneObject)

File: Maple Fighters/Servers/Game/Game/Application/Scene Objects/Portal/Portal.cs
using MathematicsHelper;
using ServerApplication.Common.Components;
using Game.Common;
using InterestManagement;
using InterestManagement.Components;

namespace Game.Application.SceneObjects
{
    public class Portal : SceneObject
    {
        public Portal(Vector2 position, Maps destinationMap) 
            : base(IdGenerator.GetId(), "Portal", new TransformDetails(position, Vector2.Zero, Direction.Left))
        {
            Components.AddComponent(new PortalInfoProvider(destinationMap));
        }
    }
}